wrappers/python/examples/roberta_training.py

```diff
@@ -244,10 +243,8 @@
             current_label[inverse_current_mask] = -100
             y.from_array(np.asfortranarray(current_label.T))
             minibatch_labels.append(y)
-        masked_data_epoch.append(minibatch_masked_data)
-        labels_epoch.append(minibatch_labels)
-    batch_masked_data.append(masked_data_epoch)
-    batch_labels.append(labels_epoch)
+    batch_masked_data.append(minibatch_masked_data)
```

Comment:
`minibatch_masked_data` is emptied on every train batch. Therefore, only the last one appears in `batch_masked_data`. Move lines 223 and 224 out of iteration over number of train batches.

Other possible solution was to switch `append` function with `extend` function in the previous version of the code:
```py
batch_masked_data.append(masked_data_epoch)
batch_labels.append(labels_epoch)
```
into
```py
batch_masked_data.extend(masked_data_epoch)
batch_labels.extend(labels_epoch)
```